quacc/settings.py

```diff
@@ -128,6 +128,38 @@ class QuaccSettings(BaseSettings):
         description="After this many seconds, Custodian will stop running and ensure that VASP writes a STOPCAR",
     )
 
+    # ---------------------------
+    # Q-Chem Settings
+    # ---------------------------
+
+    # Q-Chem Settings: Main
+    QChem_CMD: str = Field(
```

Comment:
Let's use all-caps for the global settings names.

Reply:
Sounds good, thanks for making this change!